theories/datatypes/Xreal.ec

```diff
@@ -354,7 +354,11 @@ proof. by move=> /(xler_addr z) h1 /(xler_addl y); apply xle_trans. qed.
 lemma xler_pmul2l (x:realp) : 0%rp < x => 
   forall (y z : xreal),
   rp x * y <= rp x * z <=> y <= z.
-proof. move=> hx y z; case: z => // z; case: y => // y; smt(to_realP). qed.
+proof.
```

Comment:
Just realised that this is probably best done (given appearance below) as a more generic lemma on multiplication between a positive and an extended real.

Reply:
I cannot figure out how to even express the result I want to pull out of this proof in a way that keeps it usable. So I'm resolving this comment, but happy to revisit with some guidance on how to get my head around the notations here.